torch/ao/ns/_numeric_suite_fx.py

```diff
@@ -752,6 +752,9 @@ def prepare_n_shadows_model(
     example_inputs: Any,
     qconfig_multi_mapping: QConfigMultiMapping,
     backend_config: BackendConfig,
+    exposed_prepare_function: Any=None,
```

Comment:
nit: fix lint

Reply:
would a name such as `prepare_fn_override` and `convert_fn_override`, or `custom_prepare_fn` / `custom_convert_fn` make more sense?  the `exposed` word seems ambiguous

Reply:
ok, will update